Customer Demo/Customer Demo/Screens/Setup/SetupViewController.swift

```diff
@@ -59,6 +62,13 @@ class SetupViewController: UIViewController {
 
         navigationController?.pushViewController(subscriberStatusViewController, animated: true)
     }
+    
+    func showValidationError() {
+        let alertController = UIAlertController(title: "Validation error", message: "Please fill out all fields", preferredStyle: .alert)
+        alertController.addAction(UIAlertAction(title: "OK", style: .default))
+        
+        self.present(alertController, animated: true)
```

Comment:
I don't think you need to add `self` here ;) Same in the `showError` method below.

Reply:
Removed